### PR TITLE
bsc#1042225 fix (hardware reqs for monasca server node)

### PR DESCRIPTION
**Pull request by dmpop**
@taroth21 @jgrassler Please take a look at this, and let me know if you have any feedback. Thank you!

### CODE REVIEW

File: xml/depl_require.xml [hunk 1]
```diff
@@ -1406,6 +1406,110 @@
     For storage requirements, see <xref linkend="sec.depl.req.storage.hardware.store"/>.
    </para>
   </sect2>
+
+<sect2 xml:id="sec.depl.req.hardware.monnode">
+   <title>&monnode;</title>
+   <para>
+     The Monasca agent roles, such as <literal>monasca-agent</literal> and
```
Comment (taroth21) [inline xml/depl_require.xml]: I think the information where to deploy which roles rather belongs to the Monasca barclamp section. The only piece of information I would mention here is probably the following: 
The &monnode; is a dedicated physical machine that runs the ``monasca-server``` role. It is used for &productname; Monitoring.  

In fact, even this definition (what the Monasca node is and what is it good for), should be covered in chapter 1 already (I will add it there as part of bsc#1042220 as soon as I have received some input by Johannes).

File: xml/depl_require.xml [hunk 2]
```diff
@@ -1406,6 +1406,110 @@
...
+     <literal>monasca-log-agent</literal>, should be assigned to all nodes,
+     regardless of their other roles. However, the
+     <literal>monasca-server</literal> role must be deployed on a dedicated physical machine
+     referred as &monnode;. Hardware required for the &monnode; are as follows:
```
Comment (taroth21) [inline xml/depl_require.xml]: Hardware required for -> The hardware requirements for [...]? (but I'm not sure about this).

File: xml/depl_require.xml [hunk 3]
```diff
@@ -1406,6 +1406,110 @@
...
+     </para>
+     </listitem>
+   <listitem>
+     <para>
+      20 bytes in size
+     </para>
+     </listitem>
+   </itemizedlist>
+   <para>
+     Please note that the formula provides only a rough estimation of the
```
Comment (taroth21) [inline xml/depl_require.xml]: I would put this information directly after the formula and remove the 'Please note':
The formula provides only...

File: xml/depl_require.xml (hunk 2 above)
Comment (jgrassler) [inline xml/depl_require.xml]: Yes, either "The hardware requirements for [...] are as follows" or "The following Hardware is required for [...]. Right now this is a mix of both.

File: xml/depl_require.xml [hunk 4]
```diff
@@ -1406,6 +1406,110 @@
...
+     required disk space. Depending on the number of monitored nodes, other
```
Comment (jgrassler) [inline xml/depl_require.xml]: I'd reformulate this a bit and make it more concrete: "The mix of  services which actually run on your OpenStack nodes, your cloud's usage pattern and whether any or all services have debug logging enabled may lead to higher or lower figures than what the formula yields."

File: xml/depl_require.xml (hunk 1 above)
Comment (dmpop) [inline xml/depl_require.xml]: Makes sense. Fixed.

File: xml/depl_require.xml (hunk 2 above)
Comment (dmpop) [inline xml/depl_require.xml]: Oops, my bad! Fixed.

File: xml/depl_require.xml (hunk 3 above)
Comment (dmpop) [inline xml/depl_require.xml]: Sounds sensible, but I think the assumptions and recommended values take precedence over the note. In other words, it's more important to clarify the formula's values and basis before providing additional info. Thoughts?

File: xml/depl_require.xml (hunk 4 above)
Comment (dmpop) [inline xml/depl_require.xml]: How about this, then?

_There are several factors that can affect disk space requirements. This
includes the exact combination of services that run on your OpenStack nodes,
actual cloud usage pattern, and whether any or all services have debug logging enabled._

Comment (jgrassler) [inline xml/depl_require.xml]: Yes, that looks good. Thanks!

File: xml/depl_require.xml (hunk 2 above)
Comment (jgrassler) [inline xml/depl_require.xml]: Are you sure? The diff still shows the old version ("Hardware required for the &monnode; are as follows").

File: xml/depl_require.xml (hunk 3 above)
Comment (taroth21) [inline xml/depl_require.xml]: Fine with me, too. :)

File: xml/depl_require.xml (hunk 4 above)
Comment (taroth21) [inline xml/depl_require.xml]: Fine with me, too. :)

**Pull request merged**